sqlitecluster/SQLiteNode.cpp

```diff
@@ -1082,8 +1086,9 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
         if (peer->params["Permafollower"] != "true" && !message.calc("Priority")) {
             STHROW("you're *not* supposed to be a 0-priority permafollower");
         }
-        // It's an error to have to peers configured with the same priority, except 0.
-        SASSERT(!_priority || message.calc("Priority") != _priority);
+
+        // It's an error to have to peers configured with the same priority, except 0 and -1
+        SASSERT(!_priority || _priority < 0 || message.calc("Priority") != _priority);
```

Comment:
you can ditch `!_priority` and make the next conditional `_priority <= 0`

Reply:
I think you're right about `!_priority`, but I think "less than 0" is a bad idea in the first place, because it's not accurate, it's opening up an opportunity for us to do something dumb later. I'm going to replace it with "equals -1 or equals 0 or <blah>".